### PR TITLE
Website: fix docs refering to create/load container APIs

### PR DESCRIPTION
**Pull request by micahgodbolt**
We have a lot of legacy docs with references to old APIs

### CODE REVIEW

File: docs/content/docs/build/data-modeling.md [hunk 1]
```diff
@@ -62,10 +61,10 @@ const schema = {
     dynamicObjectTypes: [ SharedCell, SharedMap ],
 }
 
-const { fluidContainer, containerServices } = await client.getContainer(/*service config*/, schema);
+const { container, services } = await client.getContainer(/*service config*/, schema);
```
Comment (sumedhb1995) [inline docs/content/docs/build/data-modeling.md:64]: I think the config here will just be the id string now, here and in the other occurrences below

Comment (micahgodbolt) [reply docs/content/docs/build/data-modeling.md:64]:
```suggestion
const { container, services } = await client.getContainer(id, schema);
```

File: docs/content/docs/build/data-modeling.md [hunk 2]
```diff
@@ -95,17 +94,16 @@ using the handle. It also demonstrates retrieving the `SharedCell` object from t
...
-    name: "example-container",
     initialObjects: {
         map: SharedMap,
     },
     dynamicObjectTypes: [ SharedCell ],
 }
 
-const { fluidContainer, containerServices } = await client.getContainer(/*service config*/, schema);
-const map = fluidContainer.initialObjects.map;
+const { container, services } = await client.getContainer(/*service config*/, schema);
```
Comment (sumedhb1995) [inline docs/content/docs/build/data-modeling.md:103]:
```suggestion
const { container, services } = await client.getContainer(id, schema);
```

File: docs/content/docs/data-structures/map.md [hunk 3]
```diff
@@ -45,15 +44,14 @@ Similarly, if you are loading an existing container, the process stays largely i
...
 ```javascript
 const schema = {
-    name: "example-container",
     initialObjects: {
         customMap: SharedMap,
     }
 }
 
-const { fluidContainer, containerServices } = await client.getContainer(/*service config*/, schema);
+const { container, services } = await client.getContainer(/*service config*/, schema);
```
Comment (sumedhb1995) [inline docs/content/docs/data-structures/map.md:52]:
```suggestion
const { container, services } = await client.getContainer(id, schema);
```

File: docs/content/docs/data-structures/map.md [hunk 4]
```diff
@@ -64,7 +62,7 @@ const schema = {
     dynamicObjectTypes: [ SharedMap ]
 }
 
-const { fluidContainer, containerServices } = await client.getContainer(/*service config*/, schema);
+const { container, services } = await client.getContainer(/*service config*/, schema);
```
Comment (sumedhb1995) [inline docs/content/docs/data-structures/map.md:65]:
```suggestion
const { container, services } = await client.getContainer(id, schema);
```

File: docs/content/docs/data-structures/map.md [hunk 5]
```diff
@@ -315,12 +312,12 @@ const schema = {
 Now, you can dynamically create additional `SharedMap` instances and store their handles into the initial map that is always provided in the container.
 
 ```javascript
-const { fluidContainer, containerServices } = await client.getContainer(/*service config*/, schema);
+const { container, services } = await client.getContainer(/*service config*/, schema);
```
Comment (sumedhb1995) [inline docs/content/docs/data-structures/map.md:315]:
```suggestion
const { container, services } = await client.getContainer(id, schema);
```

File: docs/content/docs/data-structures/map.md [hunk 6]
```diff
@@ -412,18 +409,17 @@ Whenever a new task is created, you can call `container.create` to create a new
...
 const schema = {
-    name: "example-container",
     initialObjects: {
         initialMap: SharedMap,
     },
     dynamicObjectTypes: [SharedMap]
 }
 
-const { fluidContainer, containerServices } = await client.getContainer(/*service config*/, schema);
+const { container, services } = await client.getContainer(/*service config*/, schema);
```
Comment (sumedhb1995) [inline docs/content/docs/data-structures/map.md:418]:
```suggestion
const { container, services } = await client.getContainer(id, schema);
```

**Review by skylerjokiel — APPROVED** [review]
🍁 Signing off with Sumedh's comments

**Pull request merged**